source/api/commands/route2.md

```diff
@@ -152,6 +152,74 @@ cy.route2('/users/**', (req) => {
 
 ## Relevant Types
 
+### {% fa fa-angle-right %} IncomingHttpRequest
+
+- **Description**: Allows user to intrecept the HTTP request and track or modify the original request
```

Comment:
- **Description**: Allows user to intercept the HTTP request and track or modify the original request